src/mapbox.js

```diff
@@ -76,5 +76,12 @@ export function normalizeSourceUrl(url, token, tokenParam, styleUrl) {
     urlObject.searchParams.set(tokenParam, token);
     return decodeURI(urlObject.href);
   }
-  return `https://{a-d}.tiles.mapbox.com/v4/${mapboxPath}/{z}/{x}/{y}.vector.pbf?access_token=${token}`;
+
+  if(mapboxPath === 'mapbox.satellite'){
+    return `https://api.mapbox.com/v4/${mapboxPath}/{z}/{x}/{y}@2x.webp?access_token=${token}`;
+
+  }
+
+  return `https://api.mapbox.com/v4/${mapboxPath}/{z}/{x}/{y}.vector.pbf?access_token=${token}`;
```

Comment:
According to [Mapbox documentation], the url should look like https://a.tiles.mapbox.com/v4/mapbox.mapbox-streets-v8/14/4823/6160.mvt?access_token=...

Reply:
I just investigated into the reason why I got this issue, and solved it. For a vector I reverted the changes for this url